lib/getNFTInfo.ts

```diff
@@ -110,8 +115,8 @@ export async function getMedia({
   const finalAnimation = convertIPFS(animation_url);
 
   const [animationMimeType, imageMimeType] = await Promise.all([
-    getMimeType(animation_url || ''),
-    getMimeType(image || ''),
+    getMimeType(finalAnimation),
```

Comment:
we should have been using these URLs, which had already been converted to use our IPFS gateway